lldb/source/API/SBType.cpp

```diff
@@ -586,6 +586,15 @@ lldb::TemplateArgumentKind SBType::GetTemplateArgumentKind(uint32_t idx) {
   return eTemplateArgumentKindNull;
 }
 
+SBType SBType::FindNestedType(const char *name) {
+  LLDB_INSTRUMENT_VA(this);
```

Comment:
`name` needs to be in the instrumentation macro
```
LLDB_INSTRUMENT_VA(this, name);
```

Reply:
I didn't knew this. Thank you!